prisma/schema.prisma

```diff
@@ -0,0 +1,40 @@
+generator client {
+  provider = "prisma-client-js"
+    previewFeatures = ["fullTextSearch"]
+
+}
+
+datasource db {
+  provider = "postgresql"
+  url      = env("DATABASE_URL")
+}
+
+model User {
+  id    String @id @default(uuid())
+  name  String
+  email String
+  List  List[]
+}
+
+model List {
+  id     String  @id @default(uuid())
+  name   String 
+  user   User?   @relation(fields: [userId], references: [id])
+  userId String?
+  items  Items[]
+}
+
+model Categories {
+  id    String  @id @default(uuid())
+  name  String  @unique
+  Items Items[]
+}
+
+model Items {
+  id                 String      @id @default(uuid())
+  name               String
+  Linked_to_List     List?       @relation(fields: [ListID], references: [id])
```

Comment:
@n1lsw please see Dans comments in discord about naming conventions here and address that in a fix-PR, then merge those changes into this PR from main, once the fix-PR has been merged.

Thanks!